docs/querying/operators.md

```diff
@@ -52,8 +52,9 @@ The following binary comparison operators exist in Prometheus:
 * `<=` (less-or-equal)
 
 Comparison operators are defined between scalar/scalar, vector/scalar,
-and vector/vector value pairs. By default they filter. Their behavior can be
-modified by providing `bool` after the operator, which will return `0` or `1`
+and vector/vector value pairs. By default they filter: for example, the value of
```

Comment:
This is duplicating the information right below this, and confusing as it's only mention one of the three sets of semantics.